src/test/java/com/exactpro/th2/common/builder/BuildersTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.common.builder;
+
+import org.junit.jupiter.api.Test;
+
+import com.exactpro.th2.common.grpc.ConnectionID;
+import com.exactpro.th2.common.grpc.Direction;
+import com.exactpro.th2.common.grpc.EventID;
+import com.exactpro.th2.common.grpc.MessageID;
+import com.exactpro.th2.common.message.MessageUtils;
+import com.exactpro.th2.common.schema.factory.CommonFactory;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BuildersTest {
+    private final CommonFactory commonFactory = CommonFactory.createFromArguments("-c", "src/test/resources/test_load_dictionaries");
+
```

Comment:
Please add test to check loading the bookName value from config and setting it to ID builders as default

Reply:
I added tests for 3 cases: default book, book from box.json and explicit book, please see.